mods/beds/init.lua

```diff
@@ -3,9 +3,18 @@ beds.player = {}
 beds.pos = {}
 beds.spawn = {}
 
+-- Intllib
+local S
+if minetest.get_modpath("intllib") then
+	S = intllib.Getter()
+else
+	S = function(s) return s end
```

Comment:
If you're using `@` replacements you need to use the longer more recent boilerplate function.